scripts/browserstack.py

```diff
@@ -188,7 +188,7 @@ def executeTests(appUrl, testUrl, isNightly):
         devices = [
             "Google Pixel 7-13.0",
             "Samsung Galaxy S22-12.0",
-            "Google Pixel 2-8.0",
```

Comment:
I think we want a wide range of test coverage. I'm fine with making this change, but it might harm us as well.